Tasks/2 - Prompt/promt.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	<head></head>
+	<body>
+		<script>
+		let cost = parseFloat(prompt("How much does a loaf of bread cost?"));
```

Comment:
Please change every `let` to `const` due to program is not modify this variables

Reply:
have done